airflow/providers/google/cloud/links/dataproc.py

```diff
@@ -32,6 +32,9 @@
 
 DATAPROC_BASE_LINK = BASE_LINK + "/dataproc"
 DATAPROC_JOB_LINK = DATAPROC_BASE_LINK + "/jobs/{job_id}?region={region}&project={project_id}"
+# TODO: remove DATAPROC_JOB_LOG_LINK alias in the next major release
```

Comment:
One (post merge) comment - I think we should not remove it in the next release if we do not do deprecation warning when it is used. The user will have no warnings whatsoever if we do.

Reply:
Sure. I tried to add the warning through https://github.com/apache/airflow/pull/33189 (the way how `DATAPROC_JOB_LOG_LINK` is mapped to `DATAPROC_JOB_LINK` has been changed as well)